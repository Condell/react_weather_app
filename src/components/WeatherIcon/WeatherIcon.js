import styled from "styled-components";
import PropTypes from "prop-types";
import React from "react";

const WeatherIcon = styled.img`
  min-height: ${props => props.large && "100px"};
  @media (min-width: 768px) {
    min-height: 100px;
  }
`;

export default WeatherIcon;
