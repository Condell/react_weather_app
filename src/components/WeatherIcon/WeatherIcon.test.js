import React from "react";
import { shallow } from "enzyme";
import 'jest-styled-components'
import toJson from "enzyme-to-json"

import WeatherIcon from "./WeatherIcon";


describe("WeatherIcon", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<WeatherIcon />);
  });

  it("should render", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});