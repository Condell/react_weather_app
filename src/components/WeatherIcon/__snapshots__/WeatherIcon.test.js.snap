// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WeatherIcon should render 1`] = `
.hYPZWU {
  min-height: ;
}

@media (min-width: 768px) {
  .hYPZWU {
    min-height: 100px;
  }
}

<img
  className="sc-bdVaJa hYPZWU"
/>
`;
